src/pages/ProductList/components/NameBar/NameBar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './NameBar.scss';
+
+const NameBar = () => {
+  return (
+    <div className="nameBar">
+      <div>
+        <span>All product</span>
+      </div>
+      <div className="sortBar">
+        <p className="sort">high price</p>
+        <p className="sort">low price</p>
```

Comment:
```suggestion
        <p className="higher sort">high price</p>
        <p className="lower sort">low price</p>
```
스타일 적용이 둘 다 한번에 적용되야 한다면 
* 공통으로 적용할 className과 함께 
* 목적을 설명할 수 있는 이름도 추가하면 됩니다.